addons/web/static/tests/core/field_service_tests.js

```diff
@@ -0,0 +1,209 @@
+/** @odoo-module **/
+
+import { fieldService } from "@web/core/field_service";
+import { makeTestEnv } from "@web/../tests/helpers/mock_env";
+import { registry } from "@web/core/registry";
+
+const serviceRegistry = registry.category("services");
+
+function getModelInfo(resModel) {
+    return {
+        resModel,
+        fieldDefs: serverData.models[resModel].fields,
+    };
+}
+
+let serverData;
+QUnit.module("Field Service", {
+    async beforeEach() {
+        serverData = {
+            models: {
+                tortoise: {
+                    fields: {
+                        id: { string: "ID", type: "integer" },
+                        display_name: { string: "Display Name", type: "char" },
+                        name: { string: "Name", type: "char", default: "name" },
+                        write_date: { string: "Last Modified on", type: "datetime" },
+                        age: { type: "integer", string: "Age" },
+                        location_id: { type: "many2one", string: "Location", relation: "location" },
+                    },
+                },
+                location: {
+                    fields: {
+                        id: { string: "ID", type: "integer" },
+                        display_name: { string: "Display Name", type: "char" },
+                        name: { string: "Name", type: "char", default: "name" },
+                        write_date: { string: "Last Modified on", type: "datetime" },
+                        tortoise_ids: { type: "one2many", string: "Turtles", relation: "tortoise" },
+                    },
+                },
+            },
+        };
+        serviceRegistry.add("field", fieldService);
+    },
+});
+
+QUnit.test("loadPath", async (assert) => {
```

Comment:
in general we define tests inside the definition of the module, not after (not sure how it behaves, maybe it works fine)

Reply:
It works fine